stable/mlrun-kit/values.yaml

```diff
@@ -49,7 +46,20 @@ nuclio:
       functions:
       - level: debug
         sink: myStdoutLoggerSink
+
 mlrun:
+  # set the type of filesystem to use: filesystem, s3
+  storage: filesystem
+  secrets:
+    s3:
+      accessKey: ""
+      secretKey: ""
```

Comment:
comment above each section when it is needed. e.g.: when ec2 instance is running with attached roles, does it needed? if so, what permission/roles it needs?
alternatively, if needs keys, what roles to attach on user?

Reply:
now it not tested and not use its just reference for next phase